flang/test/Lower/Intrinsics/selected_logical_kind.f90

```diff
@@ -0,0 +1,72 @@
+! REQUIRES: shell
```

Comment:
nit: is it really needed?

Reply:
Not sure. I copied it from the selected_int_kind test. I think it was added a while back for windows but I guess it is not necessary here. Let me remove it.